cmd/feedback/feedback.go

```diff
@@ -92,7 +93,7 @@ var SurveyStore = map[string]SlackSurvey{
 		Info: func(ctx context.Context, clients *shared.ClientFactory) {
 			clients.IO.PrintInfo(ctx, false, fmt.Sprintf(
 				"%s\n%s\n",
-				style.Secondary("Ask questions, submit issues, or suggest features for the SLack CLI:"),
+				style.Secondary("Ask questions, submit issues, or suggest features for the Slack CLI:"),
```

Comment:
note: Writing tests found a typo!

Reply:
@mwbrooks Whew great catch! It's nice to know this wasn't released, but darn that's an `L` for sure 😉